Demo/SPGooglePlacesAutocomplete/SPGooglePlacesAutocompleteDemoViewController.m

```diff
@@ -26,7 +29,34 @@ - (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
 }
 
 - (void)viewDidLoad {
+    [super viewDidLoad];
     self.searchDisplayController.searchBar.placeholder = @"Search or Address";
+
+
+
+    // Request use on iOS 8
+    if (floor(NSFoundationVersionNumber) <= NSFoundationVersionNumber_iOS_7_1) {
```

Comment:
I'm not sure I understand how it relates to `setShowsUserLocation:`.

As for the test itself (I'm being picky here as this is simply a demo project but I want to insist on that for the future), we should avoid testing for OS version and instead test for API availability (`respondsToSelector`).

As an exemple for this code block, you could have written:

```objective-c
[self.locationManager setDelegate:self];

[self.locationManager setDistanceFilter:kCLDistanceFilterNone];
[self.locationManager setDesiredAccuracy:kCLLocationAccuracyBest];
if ([self.locationManager respondsToSelector:@selector(requestAlwaysAuthorization)]) {
  [self.locationManager requestAlwaysAuthorization];
}
```